dataset_builders/pie/aae2/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pie-datasets>=0.6.0,<0.9.0
```

Comment:
please also increase the min version of pie-datasets to 0.8.0 (because of https://github.com/ArneBinder/pie-datasets/pull/86 which is included in this release)

Reply:
edited in 491c137